.travis.yml

```diff
@@ -51,5 +51,5 @@ jobs:
     - stage: BC checks
       php: 7.4
       env: BC_CHECK
-      post_install: wget -O roave-backward-compatibility-check.phar https://github.com/Roave/BackwardCompatibilityCheck/releases/download/4.4.0/roave-backward-compatibility-check.phar
+      install: wget -O roave-backward-compatibility-check.phar https://github.com/Roave/BackwardCompatibilityCheck/releases/download/4.4.0/roave-backward-compatibility-check.phar
```

Comment:
I couldn't find a reference to `post_install` in the docs, and the previous build, that was on a branch I wrongly created on origin failed because this step did not seem to be executed